packages/block-library/src/navigation-link/edit.js

```diff
@@ -410,14 +410,16 @@ export default function NavigationLinkEdit( {
 		},
 		{
 			allowedBlocks: ALLOWED_BLOCKS,
-			renderAppender:
+			appender:
 				( isSelected && hasDescendants ) ||
 				( isImmediateParentOfSelectedBlock &&
 					! selectedBlockHasDescendants ) ||
 				// Show the appender while dragging to allow inserting element between item and the appender.
-				hasDescendants
-					? InnerBlocks.DefaultAppender
-					: false,
+				hasDescendants ? (
+					<InnerBlocks.DefaultAppender />
```

Comment:
Does this work ok? I thought it was `InnerBlocks.DefaultBlockAppender`. It might have been resolving as `undefined` all this time.

Reply:
Good catch! Fixed here: https://github.com/WordPress/gutenberg/pull/29911/commits/67c2525c25dceef43900aeaf8fedf8b2a4d758a0